packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.test.ts

```diff
@@ -231,9 +231,33 @@ describe('createPluginActions', () => {
         field: 'dummy',
         callbackId: TEST_CALLBACK_ID,
         filename,
+        asset: {
```

Comment:
I know these are just tests, but maybe it's useful to create an asset variable to share among them to avoid duplicating the fields everywhere.

```
export const emptyAsset: {
  id: 0,
  fieldtype: 'asset',
  name: '',
  filename: '',
  meta_data: {},
  title: '',
  copyright: '',
  focus: '',
  alt: '',
  source: '',
  is_private: false,
}
```

Reply:
applied :D